site/content/en/docs/for-users/faq.md

```diff
@@ -118,17 +112,18 @@ volumes:
 
 ## How to mark job/task as completed
 
-The status is set by the user in the [Info window](user_guide.md#info) of the job annotation view.
+The status is set by the user in the [Info window](../../../docs/for-users/user-guide/top-panel/#info)
+of the job annotation view.
 There are three types of status: annotation, validation or completed.
 The status of the job changes the progress bar of the task.
 
 ## How to install CVAT on Windows 10 Home
 
-Follow this [guide](installation.md#windows-10).
+Follow this [guide](../../../docs/for-users/installation/#windows-10).
 
 ## I do not have the Analytics tab on the header section. How can I add analytics
 
-You should build CVAT images with ['Analytics' component](../../../components/analytics).
+You should build CVAT images with ['Analytics' component](https://github.com/openvinotoolkit/cvat/tree/develop/components/analytics).
```

Comment:
Why the link leads to github instead of site/..../docs/for-developers/analytics/?

Reply:
In the original document, this link leads to a section. Should there be a link to the readme file?

Reply:
Let's change it to the readme